utils/dxbottools.py

```diff
@@ -61,6 +61,11 @@ def cancelallordersbymarket(maker, taker):
       print (results)
   return
 
+def getmyordersbymarket(maker, taker):
```

Comment:
this function just duplicates `getopenordersbymarket` and will return cancelled/completed orders